src/04-modality-specific-files/08-genetic-descriptor.md

```diff
@@ -0,0 +1,116 @@
+| Field name           | Definition                                                                     |
+|----------------------|--------------------------------------------------------------------------------|
+| Genetics.Dataset     | REQUIRED. URI where data can be retrieved.                                     |
+| Genetics.Database    | OPTIONAL. URI of database where the dataset is hosted.                         |
+| Genetics.Descriptors | OPTIONAL. List of relevant descriptors (*e.g.*, journal articles) for dataset. |
+
+Example:
+
+```JSON
+{
+  "Name": "Human Connectome Project",
+  "BIDSVersion":  "1.3.0",
+  "License": "CC0",
+  "Authors": ["1st author", "2nd author"],
+  "Funding": ["P41 EB015894/EB/NIBIB NIH HHS/United States"],
+  "Genetics": {
+     "Dataset": "https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001364.v1.p1",
+     "Database": "https://www.ncbi.nlm.nih.gov/gap/",
+     "Descriptors": ["https://doi.org/10.1016/j.neuroimage.2013.05.041"]
+     }
+}
+```
+
+## Subject naming and Participants file
+
+If the same participants have different identifiers in the genetic and imaging datasets,
+the column `genetic_id` SHOULD be added to the `participants.tsv` file to associate
+the BIDS participant with a subject in the `Genetics.Dataset` referred to in the
+`dataset_description.json` file.
+
+Information about the presence/absence of specific genetic markers MAY be duplicated
+in the `participants.tsv` file by adding optional columns (like `idh_mutation` in the
+example below).
+Note that optional columns MUST be further described in an accompanying
+`participants.json` file as described in
+[Tabular files](../02-common-principles.md#tabular-files).
+
+`participants.tsv` example:
+
+```Text
+participant_id	age	sex	group	genetic_id	idh_mutation
+sub-control01	34	M	control	124587	yes
+sub-control02	12	F	control	548936	yes
+sub-patient01	33	F	patient	489634	no
+```
+
+## Genetic Information
+
+Template:
+
+```Text
+genetic_info.json
+```
+
+The `genetic_info.json` file describes the genetic information available in the
+`participants.tsv` file and/or the genetic database described in
+`dataset_description.json`.
+Datasets containing the `Genetics` field in `dataset_description.json` or the
+`genetic_id` column in `participants.tsv` MUST include this file with the following
+fields:
+
+| Field name         | Definition                                                                  | Values                                                                                                                                                                                                             |
+| :----------------- | :-------------------------------------------------------------------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
I seem to remember this colon (`:`) convention being removed from the rest of the document during PDFification. Is that the cause of the weird table formatting [here](https://2079-150465237-gh.circle-artifacts.com/0/bids-spec.pdf#43)?

Reply:
yes, very likely. @Arshitha may have background info. I think there's no harm in removing the colons here as well. The HTML readability suffers only a little, while the PDF output is repaired.

Reply:
```suggestion
|--------------------|-----------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Reply:
yes, the colons were removed from the pipe tables for generating the pdf. Also, it might be a  good idea to extend the column widths because when text in the columns are long, it could lead to overlapping between columns (in the pdf). If it'd be valuable, I could document the conventions  that I followed for pipe tables with pdf generation in mind? @sappelhoff

Reply:
indeed that'd be helpful. Perhaps as an addition to the docs you wrote in https://github.com/bids-standard/bids-specification/blob/master/pdf_build_src/README.md ?

Reply:
I looked at the pdf and there's indeed overlapping here. I don't have the permissions to the edit file. But increasing/extending the first column width by maybe 10 hyphens or so should fix it.